app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2 class="govuk-heading-m"><%= t('.page_heading') %></h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: resource, scope: resource_name, url: user_session_path(resource_name)) do |f| %>
+      <div>
```

Comment:
HTML element: `div` are redundant as the form helper will produce its own

Reply:
👍 thanks Peter, understood and have changed